### ISSUE TITLE
Very Laggy when playing large playlists

### ISSUE THREAD

**Issue by kodarfour** [MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-#3280]
**OS:** 
Linux (KDE Plasma 5.13.2)

**GPMDP Version:** 
4.6.1

**Issue Descriptions:**
This app moves relatively smoothly until I play a song in my liked songs playlist in youtube music mode (it starts to lag alot). I suspect that it is because the playlist (316 songs) is larger than most albums/youtube created playlists.

**Steps to Reproduce:**
Open app
play a single (no lag)
play a song from large playlist (lag starts)

**Comment by InfusedPolse** [comment]
I'm having the same issue. I can play and enter playlists that are quite small but playlists that are 100+ songs (maybe, I'm not entirely sure) cannot play and I cannot enter them and edit them.

**Comment by kodarfour** [comment]
This would be understandable if this didnt occur when i used it on the website through a browser

**Comment by InfusedPolse** [comment]
Yeah, I also only have this problem in GPMDP so this has to be a GPMDP problem.

**Comment by Beartravels420** [comment]
Yep does not really support 1600song playlist literally will have to restart/fiddle for an half hour before I can get it to play
